src/client.ts

```diff
@@ -9,7 +9,7 @@ import { IFinalPing } from "./interfaces/finalPing";
  * send events to the brain and received processed input-text.
  */
 export class CognigyClient {
-	private options: Options;
+	public options: Options;
```

Comment:
You can use "protected" instead of "public" if you just want to make it "visible" to the "child"-class.